build-system/server.js

```diff
@@ -302,11 +287,39 @@ app.use('/min/', function(req, res) {
   proxyToAmpProxy(req, res, /* minify */ true);
 });
 
-app.use('/examples.build/analytics.config.json', function (req, res, next) {
+app.use('/examples/analytics.config.json', function(req, res, next) {
   res.setHeader('AMP-Access-Control-Allow-Source-Origin', getUrlPrefix(req));
   next();
 });
 
+app.get('/examples/*', function(req, res, next) {
```

Comment:
ha, that was my first try. it worked great, except for those files under a sub directory, such as /examples/pwa/pwa.html